cpp/include/milvus-storage/common/fs_util.h

```diff
@@ -23,4 +23,9 @@ Result<std::shared_ptr<arrow::fs::FileSystem>> BuildFileSystem(const std::string
 
 std::string UriToPath(const std::string& uri);
 
+static constexpr int64_t DEFAULT_MAX_ROW_GROUP_SIZE = 1024 * 1024;  // 1 MB
```

Comment:
Given `ARROW_PART_UPLOAD_SIZE=10MB`, the row group size may be too small. 

If the row group size is not configurable, we'd better find out the optimal value from some benchmarks.